src/mahesh/ManagerTable.java

```diff
@@ -0,0 +1,53 @@
+package mahesh;
+
+import mahesh.base.PredefinedActions;
+import java.util.*;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class ManagerTable extends PredefinedActions{
+	WebDriver driver;
+	
+	void setUp() {
+		driver = start();
+	}
+	
+	void managerReporting() throws InterruptedException{
+		driver.findElement(By.xpath("//a[@id='demotable']")).click();
+		Thread.sleep(2000);
+		HashMap<String, Integer> reportingMap = new HashMap<String, Integer>();
+		List<WebElement> managerList = driver.findElements(By.xpath("//table[@class='table table-striped']/tbody/tr/td[4]"));
+		for(WebElement element: managerList) {
+			String managerId = element.getText();
+			reportingMap.put(managerId, reportingMap.getOrDefault(managerId, 0)+1);
```

Comment:
Good job @MaheshKJun20 
M searching for other if-else block, after that i ralized you call getOrDefault()